docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -344,33 +344,105 @@ Sample specification
-         description: "The integer value, defaulting to 1024."
+  # roles/myapp/meta/argument_specs.yml
+  ---
+  argument_specs:
+    # roles/myapp/tasks/main.yml entry point
+    main:
+      short_description: The main entry point for the myapp role.
+      options:
+        myapp_int:
+          type: "int"
+          required: false
+          default: 42
+          description: "The integer value, defaulting to 42."
+
+        myapp_str:
+          type: "str"
+          required: true
+          description: "The string value"
+
+        myapp_list:
+          type: "list"
+          elements: "str"
+          required: true
+          description: "A list of string values."
+
+        myapp_list_with_dicts:
+          type: "list"
+          elements: "dict"
+          required: false
+          default:
+            - myapp_food_kind: "meat"
+              myapp_food_boiling_required: true
+              myapp_food_preparation_time: 60
+            - myapp_food_kind: "fruits"
+              myapp_food_preparation_time: 5
+          description: "A list of dicts with a defined structure and with default a value."
+          options:
+            myapp_food_kind:
+              type: "str"
+              choices:
+                - "vegetables"
+                - "fruits"
+                - "grains"
+                - "meat"
+              required: false
+
+            myapp_food_boiling_required:
+              type: "bool"
+              required: false
+              default: false
+              description: "Whether the kind of food requires boiling before consumption."
+
+            myapp_food_preparation_time:
+              type: int
+              required: true
+              description: "Time to prepare a dish in minutes."
+
+        myapp_dict:
+          type: "dict"
+          required: false
+          default:
+            myapp_host: "bar.foo"
+            myapp_exclude_host: true
+            myapp_path: "/etc/myapp"
+          description: "A dict with a defined structure and default values."
+          options:
+            myapp_host:
+              type: "str"
+              choices:
+                - "foo.bar"
+                - "bar.foo"
+                - "ansible.foo.bar"
+              required: true
+              description: "A string value with a limited list of allowed choices."
+            
+            myapp_exclude_host:
+              type: "bool"
+              required: true
+              description: "A boolean value."
+            
+            myapp_path:
+              type: "path"
+              required: true
+              description: "A path value."
+            
+            original_name:
+              type: list
+              elements: "str"
+              required: false
+              description: "An optional list of string values."
+
+    # roles/myapp/tasks/alternate.yml entry point
+    alternate:
+      short_description: The alternate entry point for the myapp role.
```

Comment:
```suggestion
      short_description: The alternate entry point for the myapp role.
      version_added: 1.2.0
```

Reply:
There is only a `version_added` for options, it's ignored at the entrypoint level.

Reply:
antsibull-docs has been supporting it since role support was added, so it's not ignored there.